consensus/vbft/chain_store_test.go

```diff
@@ -54,7 +54,7 @@ func newChainStore() *ChainStore {
 func TestGetChainedBlockNum(t *testing.T) {
 	chainstore := newChainStore()
 	if chainstore == nil {
-		t.Error("newChainStrore error")
+		fmt.Println("newChainStore failed")
```

Comment:
???
The fix should be in newChainStore()?

Reply:
ok